<launch>
	<!-- global rosparam -->
	<param name="use_sim_time" value="true"/>

	<!-- rosbag -->
	<node pkg="rosbag" type="play" name="player" args="--clock $(env HOME)/rosbag/airsim/drone_2cam_lidar16/2022-10-10-13-32-27.bag"/>

	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find msg_conversion)/rviz_config/demo.rviz"/>

	<!-- main -->
	<node pkg="msg_conversion" type="pc_ned_to_nwu" name="pc_ned_to_nwu" output="screen">
		<remap from="/point_cloud" to="/airsim_node/drone/lidar/LidarCustom"/>
		<param name="publish_frame" type="string" value="map"/>
	</node>
</launch>
